AdminRouter/README.md

```diff
@@ -0,0 +1,14 @@
+## AdminRouter Windows agent building and testing
+
+These scripts are written to be used as [Jenkins](https://jenkins.io/) jobs in an automated CI. Some variables will be taken from the `$env`, passed down from either a [Gearman](http://gearman.org/) or another upstream Jenkins job.
```

Comment:
You can remove this line. This is how it used to work the Jenkins server. It doesn't use Gearman now. I know this is in the other README's as well, but I'll update them.

Reply:
I will can remove them

Reply:
addressed